pythonosc/dispatcher.py

```diff
@@ -25,9 +25,10 @@ def map(self, address, handler, *args):
       - args: Any additional arguments that will be always passed to the
               handlers after the osc messages arguments if any.
     """
-    # TODO: Check if we need to use a multimap instead, spec is a bit fuzzy
-    # about it...
-    self._map[address] = Handler(handler, list(args))
+    # TODO: Check the spec:
+    # http://opensoundcontrol.org/spec-1_0
+    # regarding multiple mappings
+    self._map[address].append(Handler(handler, list(args)))
 
   def handlers_for_address(self, address_pattern):
     """Return a tuple of Handler namedtuple matching the given OSC pattern."""
```

Comment:
did you forget to include the comment change? this function now yields, please update the comment accordingly.